### PR TITLE
Drone fabricator now starts with 2000 glass/metal

### PR DESCRIPTION
**Pull request by Ziiro**
[Changelogs]:
:cl:
add: Drone fabricator can now has enough materials for two drones at roundstart
/:cl:

[why]: People rarely load the damn thing, and it's sometimes in inaccessible places. (I'm looking at you, metastation.)

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -582,7 +582,6 @@
 #include "code\game\machinery\syndicatebomb.dm"
 #include "code\game\machinery\teleporter.dm"
 #include "code\game\machinery\transformer.dm"
-#include "code\game\machinery\turnstile.dm"
```
Comment (ike709) [inline tgstation.dme]: 🤔

**Comment by FantasticFwoosh** [comment]
Often on servers its deliberately unstocked because drone griefers are common. If the /tg/ code holds up the sheets can also be removed by applying a lever on it. A button or action to turn the pre-supplied drone machine on from a starting inactive state would be design wise a lot better so the crew can turn it on and ignore it without drones printing themselves off at roundstart if they want drones to exist.

**Comment by FlattestGuitar** [comment]
I don't mind this, personally. We haven't seen any drone griefers for now and drones are always nice to have for people who want to play as them. I support this myself.

**Review by FlattestGuitar — APPROVED** [review]

**Comment by SamHPurp** [comment]
So, I have seen drone griefers, a few times.

I don't think I support this.

**Comment by ike709** [comment]
Half the time I observe and play drone. Either this, or move it to robotics on all the maps so it'll hopefully get loaded.

**Comment by davethwave** [comment]
Hope this gets merged

**Comment by Ziiro** [comment]
I'm just going to move them to more obvious areas, I think.

**Pull request closed without merging**